src/main/scala/dxWDL/dx/DxFindDataObjects.scala

```diff
@@ -149,13 +149,19 @@ case class DxFindDataObjects(limit: Option[Int], verbose: Verbose) {
       nameConstraints: Vector[String],
       withInputOutputSpec: Boolean
   ): (Map[DxDataObject, DxObjectDescribe], Option[JsValue]) = {
-    var fields = Set(Field.Name, Field.Folder, Field.Size, Field.ArchivalState, Field.Properties)
+    var fields = Set(Field.Name,
+                     Field.Folder,
+                     Field.Size,
+                     Field.ArchivalState,
+                     Field.Properties,
+                     Field.Created,
+                     Field.Modified)
```

Comment:
Why created and modified are needed? It doesn't look like we are using them anywhere (?)

Reply:
They are extracted in the parseDescribe function